java/src/org/openqa/selenium/grid/node/Node.java

```diff
@@ -150,6 +150,12 @@ protected Node(Tracer tracer, NodeId id, URI uri, Secret registrationSecret) {
       post("/session/{sessionId}/se/file")
         .to(params -> new UploadFile(this, sessionIdFrom(params)))
         .with(spanDecorator("node.upload_file")),
+      get("/session/{sessionId}/file")
+        .to(params -> new DownloadFile(this, sessionIdFrom(params)))
```

Comment:
We only would like to have a prefixed `se` endpoint because this particular functionality is not part of the W3C standard. The upload one is kept in both ways for backwards compatibility reasons.

Reply:
Fixed

Reply:
No need to comment on every review. Resolving the conversation is enough.